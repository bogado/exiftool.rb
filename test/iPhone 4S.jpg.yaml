---
:source_file: /Users/mrm/Dropbox/code/exiftool/test/iPhone 4S.jpg
:exif_tool_version: 9.41
:file_name: iPhone 4S.jpg
:directory: /Users/mrm/Dropbox/code/exiftool/test
:file_size: 1954 kB
:file_modify_date: 2013-07-15 00:17:07.000000000 +00:00
:file_access_date: 2013-11-30 07:57:12.000000000 +00:00
:file_inode_change_date: 2013-07-15 00:17:07.000000000 +00:00
:file_permissions: rw-r--r--
:file_type: JPEG
:mime_type: image/jpeg
:exif_byte_order: Big-endian (Motorola, MM)
:make: Apple
:model: iPhone 4S
:orientation: Horizontal (normal)
:x_resolution: 72
:y_resolution: 72
:resolution_unit: inches
:software: 5.0.1
:modify_date: 2011:11:23 09:39:52
:y_cb_cr_positioning: Centered
:exposure_time: !ruby/object:Rational
  denominator: 6135
  numerator: 1
:f_number: 2.4
:exposure_program: Program AE
:iso: 64
:exif_version: '0221'
:date_time_original: 2011:11:23 09:39:52
:create_date: 2011:11:23 09:39:52
:components_configuration: Y, Cb, Cr, -
:shutter_speed_value: !ruby/object:Rational
  denominator: 6135
  numerator: 1
:aperture_value: 2.4
:brightness_value: 11.27442702
:metering_mode: Multi-segment
:flash: Auto, Did not fire
:focal_length: 4.3 mm
:subject_area: 1631 1223 881 881
:flashpix_version: '0100'
:color_space: sRGB
:exif_image_width: 3264
:exif_image_height: 2448
:sensing_method: One-chip color area
:exposure_mode: Auto
:white_balance: Auto
:focal_length_in35mm_format: 35 mm
:scene_capture_type: Standard
:sharpness: Normal
:gps_latitude_ref: North
:gps_longitude_ref: West
:gps_altitude_ref: Above Sea Level
:gps_time_stamp: '19:03:32'
:gps_img_direction_ref: True North
:gps_img_direction: 121.9838275
:compression: JPEG (old-style)
:thumbnail_offset: 914
:thumbnail_length: 5987
:image_width: 3264
:image_height: 2448
:encoding_process: Baseline DCT, Huffman coding
:bits_per_sample: 8
:color_components: 3
:y_cb_cr_sub_sampling: YCbCr4:2:0 (2 2)
:aperture: 2.4
:gps_altitude: 14.2 m Above Sea Level
:gps_latitude: 37.50233333
:gps_longitude: -122.47566667
:gps_position: 37.50233333 N, 122.47566667 W
:image_size: 3264x2448
:scale_factor35efl: 8.2
:shutter_speed: !ruby/object:Rational
  denominator: 6135
  numerator: 1
:thumbnail_image: (Binary data 5987 bytes)
:circle_of_confusion: 0.004 mm
:fov: 54.4 deg
:focal_length35efl: ! '4.3 mm (35 mm equivalent: 35.0 mm)'
:hyperfocal_distance: 2.08 m
:light_value: 15.8
