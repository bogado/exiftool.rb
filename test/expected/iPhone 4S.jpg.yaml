---
:file_name: iPhone 4S.jpg
:file_size: 1954 kB
:file_modify_date_civil: 2013-07-15
:file_access_date_civil: 2013-12-01
:file_inode_change_date_civil: 2013-07-15
:file_type: JPEG
:mime_type: image/jpeg
:exif_byte_order: Big-endian (Motorola, MM)
:make: Apple
:model: iPhone 4S
:orientation: Horizontal (normal)
:x_resolution: 72
:y_resolution: 72
:resolution_unit: inches
:software: 5.0.1
:modify_date: 2011:11:23 09:39:52
:modify_date_civil: 2011-11-23
:y_cb_cr_positioning: Centered
:exposure_time: !ruby/object:Rational
  denominator: 6135
  numerator: 1
:f_number: 2.4
:exposure_program: Program AE
:iso: 64
:exif_version: '0221'
:date_time_original: 2011:11:23 09:39:52
:date_time_original_civil: 2011-11-23
:create_date_civil: 2011-11-23
:components_configuration: Y, Cb, Cr, -
:shutter_speed_value: !ruby/object:Rational
  denominator: 6135
  numerator: 1
:aperture_value: 2.4
:brightness_value: 11.27442702
:metering_mode: Multi-segment
:flash: Auto, Did not fire
:focal_length: 4.3 mm
:subject_area: 1631 1223 881 881
:flashpix_version: '0100'
:color_space: sRGB
:exif_image_width: 3264
:exif_image_height: 2448
:sensing_method: One-chip color area
:exposure_mode: Auto
:white_balance: Auto
:focal_length_in35mm_format: 35 mm
:scene_capture_type: Standard
:sharpness: Normal
:gps_latitude_ref: North
:gps_longitude_ref: West
:gps_altitude_ref: Above Sea Level
:gps_time_stamp: '19:03:32'
:gps_img_direction_ref: True North
:gps_img_direction: 121.9838275
:compression: JPEG (old-style)
:thumbnail_offset: 914
:thumbnail_length: 5987
:image_width: 3264
:image_height: 2448
:encoding_process: Baseline DCT, Huffman coding
:bits_per_sample: 8
:color_components: 3
:y_cb_cr_sub_sampling: YCbCr4:2:0 (2 2)
:aperture: 2.4
:gps_altitude: 14.2 m Above Sea Level
:gps_latitude: 37.50233333
:gps_longitude: -122.47566667
:gps_position: 37.50233333 N, 122.47566667 W
:image_size: 3264x2448
:shutter_speed: !ruby/object:Rational
  denominator: 6135
  numerator: 1
:thumbnail_image: (Binary data 5987 bytes)
:light_value: 15.8
